docs/api.md

```diff
@@ -384,8 +404,8 @@ await fs.sync("/local")
 #### `pyscript.fs.unmount`
 
 Asynchronously unmount the named `path` from the browser's virtual filesystem.
-This will free up memory. A call to [`fs.sync`](#pyscriptfssync) is
-automatically made before unmounting.
+This will free up memory and allow you to re-use the path to mount a different
```

Comment:
argh ... "*will sync and free up memory*" ... it's the internal call to `fs.sync` that was off to me, they should be sure that's included in the process somehow anyway ... sorry for the confusion

Reply:
Done ;-)